virtcontainers/qemu_ppc64le.go

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2018 IBM
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package virtcontainers
+
+import (
+	"encoding/hex"
+	"fmt"
+	"os"
+	"strconv"
+
+	govmmQemu "github.com/intel/govmm/qemu"
+)
+
+type qemuPPC64le struct {
+	// inherit from qemuArchBase, overwrite methods if needed
+	qemuArchBase
+}
+
+const defaultQemuPath = "/usr/bin/qemu-system-ppc64le"
+
+const defaultQemuMachineType = QemuPseries
+
+const defaultQemuMachineOptions = "accel=kvm,usb=off"
+
+const defaultPCBridgeBus = "pci.0"
+
+var qemuPaths = map[string]string{
+	QemuPseries: defaultQemuPath,
+}
+
+var kernelParams = []Param{
```

Comment:
Are all these options really used by ppc64el? If so, I'd suggest moving this to `qemu.go` so they can be shared by amd64+ppc64el.